aten/src/ATen/native/transformers/hip/flash_attn/ck/fmha_bwd.hpp

```diff
@@ -453,4 +453,5 @@ struct fmha_bwd_traits
     bool is_deterministic;
     // TODO: padding check is inside this api
 };
+template <int Version = 2>
```

Comment:
what is this?

Reply:
When we updated CK, this line needed to be added to appease the compiler as there are now things that look for this in the newer version.